docs/api/engine.md

```diff
@@ -190,4 +191,67 @@ engine = Engine(
         { "name": "another.module.that.needs.config", "config": {"key": "value"} }
     ]
 )
-```
\ No newline at end of file
+```
+
+## Advanced instanciation
+
+For those who want to integrate tartiflette in advanced use-cases. You could be interested by owning the process of building an `Engine()`.
+
+### Why owning the cooking (building) process of tartiflette?
+
+The cooking process of tartiflette is equals to a `build` process on another librairies, it will prepare your engine to be executed. Thus, useless to say that an engine instance can't be executed without beeing cooked. The parrallele with a tartiflette meal is so close, you can't eat a tartiflette without cooking it. That's it. 
```

Comment:
Thus, it's useless to say that an engine instance can't be executed without beeing cooked.
Like the meal, you can't eat a tartiflette without cooking it first. That's it.